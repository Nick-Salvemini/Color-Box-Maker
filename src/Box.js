import React from "react";
import './Box.css';