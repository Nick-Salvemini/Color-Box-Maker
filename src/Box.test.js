import React from "react";
import { render } from "@testing-library/react";
import Box from "./Box";

describe('Box component', () => {
    it('renders without crashing', () => {
        render(<Box color="red" height={50} width={50} remove={() => { }} />);
    });

    it('matches snapshot', () => {
        const { container } = render(<Box color="red" height={50} width={50} remove={() => { }} />);
        expect(container).toMatchSnapshot();
    });
});