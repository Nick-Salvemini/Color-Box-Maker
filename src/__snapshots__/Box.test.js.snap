// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component matches snapshot 1`] = `
<div>
  <div
    style="display: block;"
  >
    <div
      style="background-color: red; height: 50px; width: 50px; display: inline-block;"
    />
    <button
      style="background-color: red;"
    >
      Remove Box
    </button>
  </div>
</div>
`;
